daxa/process/erosita/clean.py

```diff
@@ -275,45 +275,43 @@ def flaregti(obs_archive: Archive, pimin: Quantity = Quantity(200, 'eV'), pimax:
             # TODO Need to decide which file to check for here to see whether the command has already been run
             # Make the temporary directory (it shouldn't already exist but doing this to be safe)
             if not os.path.exists(temp_dir):
                 os.makedirs(temp_dir)
 
-            flaregti_cmd = "cd {d}; flaregti eventfile={ef} gtifile={gtif} pimin={pimi} pimax={pima} " \
+            flaregti_cmd = "cd {d}; flaregti eventfile={ef} pimin={pimi} pimax={pima} " \
                  "mask_pimin={mpimi} mask_pimax={mpima} xmin={xmi} xmax={xma} ymin={ymi} ymax={yma} " \
                  "gridsize={gs} binsize={bs} detml={dl} timebin={tb} source_size={ss} source_like={sl} " \
                  "fov_radius={fr} threshold={t} max_threshold={mt} write_mask={wm} mask={m} mask_iter={mit} " \
-                 "write_lightcurve={wl} lightcurve={lcf} write_thresholdimg={wti} thresholdimg={tif}" \
-                 "; mv {ogti} {gti}; mv {olc} {lc}; mv {oti} {ti}; mv {omi} {mi}" \
+                 "write_lightcurve={wl} lightcurve={lcf} write_thresholdimg={wti} thresholdimg={tif}; " \
+                 "mv {olc} {lc}; mv {oti} {ti}; mv {omi} {mi}" \
                  "; rm -r {d}"
 
-            cmd = flaregti_cmd.format(d=temp_dir, ef=evt_list_file, gtif=og_gti_name, pimi=pimin, pima=pimax,
+            cmd = flaregti_cmd.format(d=temp_dir, ef=evt_list_file, pimi=pimin, pima=pimax,
                                       mpimi=mask_pimin, mpima=mask_pimax, xmi=xmin, xma=xmax, ymi=ymin,
                                       yma=ymax, gs=gridsize, bs=binsize, dl=detml, tb=timebin, ss=source_size,
                                       sl=source_like, fr=fov_radius, t=threshold, mt=max_threshold, wm=write_mask,
                                       m=og_maskimg_name, mit=mask_iter, wl=write_lightcurve, lcf=og_lc_name,
-                                      wti=write_thresholdimg, tif=og_thresholdimg_name, ogti=og_gti_name,
-                                      gti=gti_path, olc=og_lc_name, lc=lc_path, oti=og_thresholdimg_name,
-                                      ti=threshold_path, omi=og_maskimg_name, mi=maskimg_path)
+                                      wti=write_thresholdimg, tif=og_thresholdimg_name, olc=og_lc_name, lc=lc_path, 
+                                      oti=og_thresholdimg_name, ti=threshold_path, omi=og_maskimg_name, mi=maskimg_path)
+
             # Now store the bash command, the path, and extra info in the dictionaries
             miss_cmds[miss.name][obs_id] = cmd
             miss_final_paths[miss.name][obs_id] = final_paths
-            miss_extras[miss.name][obs_id] = {'gti_path': gti_path, 'lc_path': lc_path, 'threshold_path': threshold_path,
+            miss_extras[miss.name][obs_id] = {'lc_path': lc_path, 'threshold_path': threshold_path,
                                               'maskimg_path': maskimg_path}
 
     # This is just used for populating a progress bar during the process run
```

Comment:
The process_message doesn't need 'eROSITA' in it - just trying to make it shorter so it doesn't make TQDM's progress bars behave strangely. We should maybe shorted the 'pretty name' of eROSITACalPV also.